### PR TITLE
Updated the TOC page with the final results of the 2021 TOC election

### PR DESCRIPTION
**Pull request by geekygirldawn**
@julz @dprotaso @evankanderson - please make sure all of your info looks good before we merge this. Also, congratulations! 🎉

### CODE REVIEW

**Comment by geekygirldawn** [comment]
/assign @vaikas @bsnchan @thisisnotapril 
Can one of you merge this?

File: TECH-OVERSIGHT-COMMITTEE.md [hunk 1]
```diff
@@ -92,11 +92,13 @@ during the community feedback period at the end of each meeting.
...
-| <img width="30px" src="https://github.com/markusthoemmes.png"> | Markus Thömmes | Red Hat | [@markusthoemmes](https://github.com/markusthoemmes) | 2020-06-09 | 2022     |
-| <img width="30px" src="https://github.com/rhuss.png">          | Roland Huß     | Red Hat | [@rhuss](https://github.com/rhuss)                   | 2021-02-16 | 2022     |
+| &nbsp;                                                         | Member           | Company | Profile                                              | Term Start | Term End |
+| -------------------------------------------------------------- | ---------------- | ------- | ---------------------------------------------------- | ---------- | --------
+| <img width="30px" src="https://github.com/dprotaso.png">       | Dave Protasowski | VMware  | [@dprotaso](https://github.com/dprotaso)             | 2021-05-26 | 2022     |
+| <img width="30px" src="https://github.com/evankanderson.png">  | Evan Anderson    | VMware  | [@evankanderson](https://github.com/evankanderson)   | Bootstrap  | 2023     |
```
Comment (vaikas) [inline TECH-OVERSIGHT-COMMITTEE.md:98]: Since this is technically a new term for Evan, should we update Bootstrap to this election date?

Comment (vaikas) [reply TECH-OVERSIGHT-COMMITTEE.md:98]: @geekygirldawn wdyt?

**Comment by vaikas** [comment]
I'm going to merge this and we can revisit the term start, since I can see both be correct and everybody else is aok with it.
/lgtm
/approve

**Pull request merged**